src/TextField/TextFieldLabel.js

```diff
@@ -35,6 +55,11 @@ const propTypes = {
    * Override the inline-styles of the root element.
    */
   style: PropTypes.object,
+
```

Comment:
Looks like we have an issue with https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md 😨